bower.json

```diff
@@ -2,9 +2,9 @@
   "lib": {
     "name": "bower-rails generated lib assets",
     "dependencies": {
-      "angular": "v1.4.7",
-      "angular-route": "v1.4.x",
-      "angular-resource": "v1.4.8",
+      "angular": "=1.4.9",
```

Comment:
Technically speaking, they are the exact same. They both lock the dependency into a specific version. See semvar information [here](https://github.com/npm/node-semver).